qwc-front.php

```diff
@@ -60,6 +60,27 @@ function qwc_add_filters_front() {
 	//foreach ( $url_filters as $name => $priority ) {
 	//	add_filter( $name, 'qtranxf_convertURL', $priority );
 	//}
+
+	/* Fix the product categories and tags (displayed above the "additional informations" tab) */
+	add_filter( 'wp_get_object_terms', function ( $terms ) {
+		foreach ( $terms as $term ) {
+			if ( $term->taxonomy == 'product_cat' || $term->taxonomy == 'product_tag' ) {
```

Comment:
The line `add_filter('wp_get_object_terms', 'qtranxf_useTermLib',0);` was always in q-. Does not it work correctly?